tests/ttnn/unit_tests/operations/test_moreh_logsoftmax.py

```diff
@@ -65,37 +183,31 @@ def test_logsoftmax_for_dim_hw(shape_dim, compute_kernel_options, device):
         [[2, 3, 32 * 4, 32 * 5], 2],
     ),
 )
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        ttnn.bfloat16,
+    ],
+)
 @pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
-def test_logsoftmax_large_algorithm_for_dim_hw(shape_dim, compute_kernel_options, device):
-    device.enable_program_cache()
-
+def test_logsoftmax_large_algorithm_for_dim_hw(shape_dim, dtype, compute_kernel_options, device):
     shape, dim = shape_dim
```

Comment:
You should paramatrize the layout param instead of hardcode it to tile layout.

Reply:
The old tests already hardcoding the layout so I think the operation only supports tile layout. But if it's really necessary then I'll make the change. What do you think? @hschoi4448

Reply:
In Moreh, since only the tile layout is planned to be provided, there may not be a need to parameterize the layout.
There are only two cases where row_major is implemented:
1. When it was implemented before the decision was made to use tile layout.
2. When it was implemented initially as row_major because it was difficult to implement directly using tile layout.

Reply:
Thank you for the confirmation